app/src/main/java/org/simple/clinic/home/patients/illustration/DayOfMonth.kt

```diff
@@ -0,0 +1,30 @@
+package org.simple.clinic.home.patients.illustration
+
+import com.squareup.moshi.FromJson
+import com.squareup.moshi.ToJson
+import org.simple.clinic.util.RoomEnumTypeConverter
+import org.threeten.bp.Month
+import java.util.Locale
+
+data class DayOfMonth(
```

Comment:
We should add validation for the combination of day and month values. Feel free to do this in a later PR.

Reply:
Will add this later on